index.html

```diff
@@ -1,97 +1,164 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <!--Metadata-->
-    <meta charset="UTF-8"/>
-    <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
-    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
-    <!--Styles-->
-    <link rel="stylesheet" href="style.css"/>
-    <!--Fonts-->
-    <link rel="preconnect" href="https://fonts.googleapis.com">
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-    <link href="https://fonts.googleapis.com/css2?family=Rubik:wght@300;400;500;600&display=swap" rel="stylesheet">
-    <!--Scripts-->
+  <head>
+    <link rel="stylesheet" href="style.css" />
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
-    <script src="js.js"></script>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Rubik:wght@300;400;500;600&display=swap"
+      rel="stylesheet" />
     <title>Document</title>
-</head>
-<body>
-<header class="header">
-    <div class="header__container container">
-        <picture class="header__logo--wrapper">
-            <button class="header__burger__button">
-                <img class="header__burger__icon" src="/img/burger.svg">
-            </button>
-            <source class="header__logo__source" media="(max-width: 1280px)" srcset="/img/probuksir_logo_min.svg">
-            <img class="header__logo" src="/img/probuksir_logo.svg" alt="logo">
-        </picture>
-        <nav class="header__nav">
-            <a class="header__nav__item active" href="Home">Главная</a>
-            <a class="header__nav__item" href="">Услуги</a>
-            <a class="header__nav__item" href="">Цены</a>
-            <a class="header__nav__item" href="">Контакты</a>
-            <a class="header__nav__item" href="">Статьи</a>
-            <a class="header__nav__item" href="">Юр.лицам</a>
-        </nav>
-        <div class="header__contact">
-            <div class="header__contact__item header__contact--phone">
-                <a href="tel:+7(919)777-90-09">+7 (919) 777-90-09</a>
-                <span>24/7 Круглосуточно</span>
-            </div>
-            <a class="header__contact__item header__contact--icon" href="https://wa.me/+7(919)777-90-09">
-                <img src="img/whatsapp.svg" alt="WhatsApp">
-                <span>Чат WhatsApp</span>
-            </a>
-            <a class="header__contact__item header__contact--icon" href="#callback">
-                <img src="/img/callback.svg" alt="Заказ звонка">
-                <span>Заказ Звонка</span>
-            </a>
+  </head>
+  <body>
+    <div class="content">
+      <div class="content__list">
+        <div class="content__list__item">
+          <a href="#" class="content__list__item__link"></a>

```

Comment:
Мне кажется что пустой тег `<a>` должен как то использоваться. Мы можем див `<div class="content__list__item">` переделать в тег `<a>` чтобы вся область этой карточки была кликабельна. Как думаешь?

```suggestion
        <a class="content__list__item" href="#">

```

Reply:
Это так же относится и ко всем остальным карточкам)